src/gcovr/formats/gcov/parser/text.py

```diff
@@ -443,14 +454,33 @@ class _ParserState(NamedTuple):
     deferred_functions: list[_FunctionLine] = []
     function_name: Optional[str] = None
     function_specialization: bool = False
+    last_linecov: Optional[LineCoverage] = None
     linecov_list: list[LineCoverage] = []
     block_id: Optional[int] = None
     is_recovering: bool = False
     previous_state: Optional["_ParserState"] = None
 
+    @classmethod
+    def new(cls, **kwargs: Any) -> "_ParserState":
```

Comment:
Wouldn't this be better named 'reset' to express the comment and intent?

Or the comment should not use the word 'reset', but something like 'return a newly initialized ParserState object'?